app/views/smart_proxies/plugins/_dhcp_subnets.html.erb

```diff
@@ -0,0 +1,69 @@
+<ul id="proxy-dhcp-tab" class="nav nav-tabs nav-tabs-pf">
+  <li class="active"><a href="#dhcp_general"><%= _('General') %></a></li>
+  <% subnets.each do |subnet| %>
+    <li><a href="#<%= subnet.network.parameterize %>"><%= subnet_label subnet %></a></li>
+  <% end %>
+</ul>
+<div id="proxy-dhcp-tab-content" class="tab-content">
+  <div class="tab-pane active in" id="dhcp_general">
+    <div class="row">
+      <%= show_feature_version('dhcp') %>
+    </div>
+
+    <div class="container-fluid container-cards-pf">
+      <div class="row row-cards-pf">
+        <% subnets.in_groups_of(2).each do |subnets| %>
+          <% subnets.each do |subnet| %>
+            <div class="col-md-4">
+              <div class="card-pf card-pf-aggregate-status card-pf-accented">
+                <h2 class="card-pf-title">
+                  <a href="#<%= subnet.network.parameterize %>">
+                    <%= subnet_label(subnet) %>
+                    <span class="card-pf-aggregate-status-count">
+                      <%= Host.joins(:interfaces => :subnet).where(:subnets => {:dhcp_id => @smart_proxy.id, :network => subnet.network }).count %>
+                    </span>
+                    <%= _('Hosts') %>
+                  </a>
+                </h2>
+                <div class="card-pf-body">
+                  <p class="card-pf-aggregate-status-notifications">
+                    <a href="#<%= subnet.network.parameterize %>_reservation">
+                      <span class="fa fa-database" data-toggle="tooltip" title="<%= _('View Reservations') %>">
+                        <%= subnet.reservations.count %>
+                        <%= _('Reservations') %>
+                      </span>
+                    </a>
+                    <a href="#<%= subnet.network.parameterize %>_leases">
+                      <span class="fa fa-laptop" data-toggle="tooltip" title="<%= _('View Leases') %>">
```

Comment:
The text shouldn't go in the `span` I think. This is causing the number of leases and reservations to show up with Font awesome letters, not Patternfly. See:

![screenshot from 2016-01-27 12-25-48](https://cloud.githubusercontent.com/assets/598891/12612169/22871350-c4f1-11e5-86d2-65b7966c62d1.png)
![screenshot from 2016-01-27 12-25-33](https://cloud.githubusercontent.com/assets/598891/12612201/495ddc8e-c4f1-11e5-8ab9-72ffbfd5b549.png)